src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -418,34 +418,24 @@ void
 FinishRemoteTransactionAbort(MultiConnection *connection)
 {
 	RemoteTransaction *transaction = &connection->remoteTransaction;
-	PGresult *result = NULL;
 	const bool dontRaiseErrors = false;
-	const bool isNotCommit = false;
 
-	result = GetRemoteCommandResult(connection, dontRaiseErrors);
-
-	if (!IsResponseOK(result))
+	if (transaction->transactionState == REMOTE_TRANS_2PC_ABORTING)
 	{
-		ReportResultError(connection, result, WARNING);
-		MarkRemoteTransactionFailed(connection, dontRaiseErrors);
-
-		if (transaction->transactionState == REMOTE_TRANS_2PC_ABORTING)
+		PGresult *result = GetRemoteCommandResult(connection, dontRaiseErrors);
+		if (!IsResponseOK(result))
 		{
+			const bool isNotCommit = false;
```

Comment:
While we're changing this, we should maybe use `isCommit`. These double negations are confusing.